edison-node.js

```diff
@@ -0,0 +1,76 @@
+/*
+* Gulp Common - Microsoft Sample Code - Copyright (c) 2016 - Licensed MIT
+*/
+'use strict';
+
+var args = require('get-gulp-args')();
+var fs = require('fs');
+
+/**
+ * Main entry point for all Rapberry Pi Node configuration.
+ * @param {object} gulp     - Gulp instance
+ * @param {object} options  - Raspberry Pi Node Specific options
+ */
+function initTasks(gulp, options) {
+  var runSequence = require('run-sequence').use(gulp);
+  var all = require('./all.js')(options);
+  var config = all.getConfig();
+
+  // stick config into gulp object
+  gulp.config = config;
+
+  if (typeof all.gulpTaskBI === 'function') {
+    all.gulpTaskBI(gulp, 'nodejs', 'Edison', ((options && options.appName) ? options.appName : 'unknown'));
+  }
+
+  gulp.task('init', 'Initializes sample', function (cb) {
+
+    if (options.configPostfix && options.configTemplate) {
+      all.updateGlobalConfig(options.configPostfix, options.configTemplate);
+    }
+
+    cb();
+  })
+
```

Comment:
On newly flashed device, the node version is 4.4. So I think we don't need 'install-tools' task. 

Will consider merging this js with 'raspberrypi-node' file later.